docs/content/ingestion/batch-ingestion.md

```diff
@@ -82,6 +82,8 @@ instance of a Druid [overlord](../design/indexing-service.html). A sample task i
 |hadoopDependencyCoordinates|A JSON array of Hadoop dependency coordinates that Druid will use, this property will override the default Hadoop coordinates. Once specified, Druid will look for those Hadoop dependencies from the location specified by `druid.extensions.hadoopDependenciesDir`|no|
 |classpathPrefix|Classpath that will be pre-appended for the peon process.|no|
 
+also note that, druid automatically computes the classpath for hadoop job containers that run in hadoop cluster. But, in case of conflicts between hadoop and druid's dependencies, you can manually specify the classpath by setting `druid.extensions.hadoopContainerDruidClasspath` property. See the extensions config in [base druid configuration](../configuration/index.html).
```

Comment:
other-hadoop.md is about setting up a different hadoop and unrelated to this. this is to be used when your MR containers failing due to a conflict in druid dependencies and hadoop's own dependencies (or if you have hdfs-storage and hadoop jars on main druid process classpath like we do leading to duplicate versions of jersey, guava etc showing up... our hadoop versions do not live in mvn or any other conflicts). 
basically when druid computed classpath for hadoop job does not work for you for whatever reason, you can override same by this setting.